src/helpers/time_ago_helper.cr

```diff
@@ -0,0 +1,28 @@
+module Helpers::TimeAgoHelper
+  extend self
+
+  def time_ago_in_words(time)
+    diff = (Time.now - time.not_nil!)
+
+    case diff
+    when 0.seconds..1.minute
+      "just now"
+    when 1.minute..2.minutes
+      "a minute ago"
+    when 2.minutes..1.hour
+      "#{(diff / 60).to_i} minutes ago"
+    when 1.hour..2.hours
+      "an hour ago"
+    when 2.hours..1.day
+      "#{((diff / 60) / 60).to_i} hours ago"
+    when 1.day..2.days
+      "a day ago"
+    when 2.days..1.week
+      "#{((diff) / (60 * 60 * 24)).to_i} days ago"
+    when 1.week..2.weeks
+      "a week ago"
+    else
+      "#{((diff) / (60 * 60 * 24 * 7)).to_i} weeks ago"
```

Comment:
Can we continue with `month` and `year`? I don't think something like `112 weeks ago` is useful :)

Reply:
i'm having some trouble using months in ranges

```crystal
when 2.weeks..1.month
  "#{((diff) / (60 * 60 * 24 * 7)).to_i} weeks ago"
end
```

```
in /opt/crystal/src/range.cr:211: no overload matches 'Time::Span#<' with type Time::MonthSpan
Overloads are:
 - Comparable(T)#<(other : T)

      @begin <= value < @end
                ^
```

Reply:
maybe use `4.weeks` instead ?

Reply:
Using plain weeks may cause some edge cases to output a wrong time ago, for example, using ```20.years.ago``` outputs ```21 years ago```. I'll try to find a way to use [Time::MonthSpan](https://crystal-lang.org/api/0.23.1/Time/MonthSpan.html) (that is separate from [Time::Span](https://crystal-lang.org/api/0.23.1/Time/Span.html) later.

Reply:
```diff.minutes``` is much prettier than ```(diff / 60).to_i```, thanks! :smiley: